staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -999,6 +999,8 @@ message ContainerStatus {
     repeated Mount mounts = 14;
     // Log path of container.
     string log_path = 15;
+    // Resource limits configuration of the container.
+    ContainerResources resources = 16;
```

Comment:
Can you also update the v1/api.proto to keep both the versions the same? We will drop v1alpha2 once containerd is updated to support the v1.

Reply:
Thanks for noticing this issue. It is fixed now.